saleor/payment/gateways/adyen/plugin.py

```diff
@@ -168,6 +181,17 @@ class AdyenGatewayPlugin(BasePlugin):
             ),
             "label": "Enable native 3D Secure",
         },
+        "apple-pay-cert": {
+            "type": ConfigurationTypeField.SECRET_TEXT,
+            "help_text": (
+                "Follow the Adyen docs related to activating the apple pay for the "
```

Comment:
"Apple Pay" is a proper name so please fix all occurrences to use proper capitalization.